### PR TITLE
feat: add sfs MD

### PR DESCRIPTION
**Pull request by deer-hang**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
add sfs MD information
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ SFS Turbo CSI 插件支持共享卷的创建、挂载和扩容。共享卷是一
 
 更多详情请参考[弹性文件服务Turbo CSI插件](./docs/sfsturbo/sfsturbo.md)
 
+### 弹性文件服务（SFS容量型）插件
```
Comment (chengxiangdong) [inline README_CN.md:21]:
```suggestion
###  SFS 容量型插件
```

Comment (chengxiangdong) [inline README_CN.md:21]: 专业缩写名词不做中文翻译，参考如下：
`弹性文件服务Turbo插件` -> `SFS Turbo 插件`
`云硬盘插件` -> `云硬盘（EVS）插件`


其他存储的中文名：
`SFS 3.0 容量型插件`、`对象存储服务（OBS）插件`

File: README_CN.md [hunk 2]
```diff
@@ -18,6 +18,12 @@ SFS Turbo CSI 插件支持共享卷的创建、挂载和扩容。共享卷是一
...
+
+SFS CSI 插件支持创建共享卷及添加共享规则。
+
+更多详情请参考[弹性文件服务插件](./docs/sfs/sfs.md)
```
Comment (chengxiangdong) [inline README_CN.md:25]: ditto

File: README_CN.md (hunk 1 above)
Comment (deer-hang) [reply README_CN.md:21]: done

Comment (deer-hang) [reply README_CN.md:21]: 收到

File: README_CN.md (hunk 2 above)
Comment (deer-hang) [reply README_CN.md:25]: done

**Pull request merged**